sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_attribution_v1/schema.yaml

```diff
@@ -68,3 +68,9 @@ fields:
 - mode: NULLABLE
   name: attributed
   type: BOOLEAN
+- mode: NULLABLE
```

Comment:
maybe it would be defining `description` field here along with the description for any field which might not be self explanatory? For example, `install_source` what does this actually mean?